pkgs/development/haskell-modules/configuration-ghc-8.4.x.nix

```diff
@@ -0,0 +1,764 @@
+      url    = "https://github.com/sol/stringbuilder";
+      rev    = "4a1b689d3c8a462b28e0d21224b96165f622e6f7";
+      sha256 = "0h3nva4mwxkdg7hh7b7a3v561wi1bvmj0pshhd3sl7dy3lpvnrah";
+    };
+    doCheck         = false;
+    jailbreak       = true;
+  });
+
+  tagged = overrideCabal super.tagged (drv: {
+    src = pkgs.fetchgit {
+      url    = "https://github.com/ekmett/tagged";
+      rev    = "d86a2109773bd79adecf78ed49f85316ff0ae2ad";
+      sha256 = "1hfq660rc05ljqhnr65n01kmxvyfad7widwg5qn83a74v41qbhap";
+    };
+    doCheck         = false;
+    jailbreak       = true;
+    editedCabalFile = null;
+    revision        = null;
+  });
+
+  th-desugar = overrideCabal super.th-desugar (drv: {
+    src = pkgs.fetchgit {
+      url    = "https://github.com/goldfirere/th-desugar";
+      rev    = "879edfa2a09fb5a829b40ccb55de960398bf8cea";
+      sha256 = "1p3aac87sfrrs9p4llb63qb891slx2mjz7vf255lvvgcfkhqrcrp";
+    };
+    doCheck         = false;
+    jailbreak       = true;
+  });
+
+  unordered-containers = overrideCabal super.unordered-containers (drv: {
+    src = pkgs.fetchgit {
+      url    = "https://github.com/tibbe/unordered-containers";
+      rev    = "60ced060304840ed0bf368249ed6eb4e43d4cefc";
+      sha256 = "10sa6h6cvhrsg8yqg23dg07q28liczgqwa084zyrkpifsg0j3zhq";
+    };
+    doCheck         = false;
+    jailbreak       = true;
+  });
+
+  websockets = overrideCabal super.websockets (drv: {
+    src = pkgs.fetchgit {
+      url    = "https://github.com/jaspervdj/websockets";
+      rev    = "11ba6d15cf47bace1936b13a58192e37908b0300";
+      sha256 = "1swphhnqvs5kh0wlqpjjgx9q91yxi6lasid8akdxp3gqll5ii2hf";
+    };
+
+  });
+
+
+# Unmerged
+
+  blaze-builder = overrideCabal super.blaze-builder (drv: {
+    src = pkgs.fetchgit {
+      url    = "https://github.com/lpsmith/blaze-builder";
+      rev    = "b7195f160795a081adbb9013810d843f1ba5e062";
+      sha256 = "1g351fdpsvn2lbqiy9bg2s0wwrdccb8q1zh7gvpsx5nnj24b1c00";
+    };
+    doCheck         = false;
+    jailbreak       = true;
+  });
+
+  bytestring-trie = overrideCabal super.bytestring-trie (drv: {
+    src = pkgs.fetchgit {
+      url    = "https://github.com/wrengr/bytestring-trie";
+      rev    = "e0ae0cb1ad40dedd560090d69cc36f9760797e29";
+      sha256 = "1jkdchvrca7dgpij5k4h1dy4qr1rli3fzbsqajwxmx9865rgiksl";
+    };
+    doCheck         = false;
+    jailbreak       = true;
+    libraryHaskellDepends = drv.libraryHaskellDepends ++ [ self.data-or ];
+  });
+
+  cereal = overrideCabal super.cereal (drv: {
+    src = pkgs.fetchgit {
+      url    = "https://github.com/GaloisInc/cereal";
+      rev    = "c2f233a3dd19655edf40841f91848ea1e78978a6";
+      sha256 = "1gfj9vnrk38d3a02wz2w1x9vcdr8k50g0f8djcgcqqkdmh1qrzyh";
+    };
+    doCheck         = false;
+    jailbreak       = true;
+  });
+
+  conduit-extras = overrideCabal super.conduit-extras (drv: {
+    src = pkgs.fetchgit {
+      url    = "https://github.com//conduit-extras";
+      rev    = "";
```

Comment:
This doesn't seem like a good idea.

Reply:
Whoops, thank you!  It's a bit odd the resulting package set works here.. Hmm.